### ISSUE TITLE
RTL 504 and grpc message size exceeded for node with too many payments.

### ISSUE THREAD

**Issue by miketwenty1** [Ride-The-Lightning/RTL#909]
**Describe the bug**
max_payments equal to 1,000,000 too high for grpc message
If you have too many payments, RTL will bug out.

**To Reproduce**
1. login to RTL
2. do nothing, wait.. 
3. get error / crash

**Screenshots**

**Your environment**
* Version of `RTL` 
0.11.2
* Version of `lnd`
v0.14.1-beta

**Additional context**
```
[2021-12-17T03:20:50.688Z] ERROR: Payments => All Lightning Transactions Error: {"name":"StatusCodeError","statusCode":429,"message":"429 - {\"code\":8,\"message\":\"grpc: received message larger than max (377418897 vs. 209715200)\",\"details\":[]}","error":{"code":8,"message":"grpc: received message larger than max (377418897 vs. 209715200)","details":[]},"options":{"url":"https://nodehostname.io:8080/v1/payments?max_payments=1000000&index_offset=0&reversed=true","rejectUnauthorized":false,"json":true,"headers":{},"method":"GET","qs":{},"simple":true,"resolveWithFullResponse":false,"transform2xxOnly":false},"response":{"statusCode":429,"body":{"code":8,"message":"grpc: received message larger than max (377418897 vs. 209715200)","details":[]},"headers":{"content-type":"application/json","grpc-metadata-content-type":"application/grpc","date":"Fri, 17 Dec 2021 03:20:50 GMT","content-length":"102","connection":"close"},"request":{"uri":{"protocol":"https:","slashes":true,"auth":null,"host":"nodehostname:8080","port":"8080","hostname":"nodehostname","hash":null,"search":"?max_payments=1000000&index_offset=0&reversed=true","query":"max_payments=1000000&index_offset=0&reversed=true","pathname":"/v1/payments","path":"/v1/payments?max_payments=1000000&index_offset=0&reversed=true","href":"https://nodehostname:8080/v1/payments?max_payments=1000000&index_offset=0&reversed=true"},"method":"GET","headers":{"accept":"application/json"}}}}
```

You can see here in the logs you are trying to do "max_payments=1000000" this is too high and you will get an error if a node has more than a few hundred thousand successful payments. I safer number is setting this to 100,000 or less.

**Comment by miketwenty1** [comment]
https://github.com/Ride-The-Lightning/RTL/pull/904#issuecomment-996432280

**Comment by saubyk** [comment]
This issue has been addressed with release v0.12.0. Closing this ticket.
